programs/aes/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ set(executables
 foreach(exe IN LISTS executables)
     add_executable(${exe} ${exe}.c $<TARGET_OBJECTS:mbedtls_test>)
     target_link_libraries(${exe} ${mbedcrypto_target})
+    target_include_directories(${exe} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/../../tests/include)
```

Comment:
Since sample programs are no longer using `mbedtls_test_rnd_std_rand`, including `tests/include` is no longer necessary.

Reply:
Two lines above we're linking with the objects in `mbedtls_test` unconditionally, so it seems more consistent to me to also put  `tests/include` on the include path unconditionally. It's also more consistent with Make which has `-Itests/include` for all programs.

Also, I don't think having this directory on the input path even for programs that don't happen to use it now hurts in any way.